infra/README.md

```diff
@@ -1,23 +1,26 @@
-# [Engineering Playbook](../README.md) / Infrastructure/Deployment
+# [Engineering Playbook](../README.md) / Infrastructure and Deployment
 
 ## Overview
 
-Infrastructure/Deployment is rilly rilly important - much too important for humans to do. If at all possible - automate it.
```

Comment:
the robot revolution of the future is sorely disappointed in this change of language, I'm sure

Reply:
Sob. Was desperately channelling Tank Girl when I wrote this.